### PR TITLE
Removed vision ga samples

### PR DESCRIPTION
**Pull request by munkhuushmgl**
- [X] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: vision/automl/README.md [hunk 1]
```diff
@@ -1,100 +1,3 @@
...
-```
-### Predict API
-
-```
-mvn exec:java -Dexec.mainClass="com.google.cloud.vision.samples.automl.PredictionApi" \
--Dexec.args="[model-id] ./resources/dandelion.jpg 0.7"
-```
+# Vision automl samples has been moved to java-automl repo
 
+[Java AutoML repo](https://github.com/googleapis/java-automl/tree/master/samples) Java samples.
```
Comment (averikitsch) [inline vision/automl/README.md:3]:
```suggestion
These samples have moved to [googleapis/java-automl](https://github.com/googleapis/java-automl/tree/master/samples).
```

Comment (munkhuushmgl) [reply vision/automl/README.md]: I will apply this to all other READMEs

File: vision/automl/README.md [hunk 2]
```diff
@@ -1,100 +1,3 @@
...
-mvn exec:java-Dexec.mainClass="com.google.cloud.vision.samples.automl.ModelApi" \
--Dexec.args="delete_model [model-id]"
-```
-### Predict API
-
-```
-mvn exec:java -Dexec.mainClass="com.google.cloud.vision.samples.automl.PredictionApi" \
--Dexec.args="[model-id] ./resources/dandelion.jpg 0.7"
-```
+# Vision automl samples has been moved to java-automl repo
```
Comment (averikitsch) [inline vision/automl/README.md:1]:
```suggestion
# Google AutoML Vision Samples
```

**Review by averikitsch — CHANGES_REQUESTED** [review]
Please update all the readmes - see comments for example

File: vision/automl/README.md (hunk 2 above)
Comment (munkhuushmgl) [reply vision/automl/README.md]: > Please update all the readmes - see comments for example

Will do, thanks for quick response.

**Review by averikitsch — APPROVED** [review]
Thanks Mike!

**Pull request merged**